### PR TITLE
Linting and other cleanup

### PR DESCRIPTION
**Pull request by ghtdak**
When going through code for the first time, I often clean it up.  Some projects hate this because it breaks blame.  I, OTOH, prefer PEP-8 (or reasonably close).

Tested with trial and have no errors.

### CODE REVIEW

**Comment by meejah** [comment]
Thanks for the PR. You can run "tox" to build against the various environments and dependencies.

File: test/test_endpoints.py [hunk 1]
```diff
@@ -430,17 +430,20 @@ def test_endpoint_connect_tor(self, private_tor, global_tor, system_tor):
...
 
-    def dataReceived(self, data):
+    @staticmethod
+    def dataReceived(data):
         print "DATA", data
 
-    def connectionLost(self, reason):
+    @staticmethod
+    def connectionLost(reason):
         print "LOST", reason
```
Comment (meejah) [inline test/test_endpoints.py]: I know these methods don't use self but defining them as @staticmethod kind of goes against the interface definition...not entirely sold ;)

However I am interested in "more PEP8" in general :)

Comment (ghtdak) [inline test/test_endpoints.py]: I'm also not sure why static methods are "a thing".  Not really sure why that's a thing.  Easy to turn that off

I'll broaden the coverage.

**Comment by ghtdak** [comment]
Hmmm.  Looks like there are more than a few mistakes.  I'm gonna pull this for the time being.

**Pull request closed without merging**